### ISSUE TITLE
Unable to override autoComplete="off"

### ISSUE THREAD

**Issue by jakubgawlikowski** [paypal/downshift#457]
* `downshift` version: 1.31.15
* `node` version: 9.7.0
* `npm` (or `yarn`) version: yarn 1.6.0

Relevant code or config

```javascript
import Downshift from 'downshift';
...
render() {
...
    return (
      <Downshift
...
        autoComplete="asdf"
        render={({ getRootProps, ...attrs }) => (
          <DropdownRenderer
            itemValueKey={itemValueKey}
            {...getRootProps({ refKey: 'rootRef' })}
            {...rest}
            {...attrs}
          />
        )}
      />
    );
```

What you did:
I tried to override autocomplete="off" in order to solve this issue: https://stackoverflow.com/questions/12374442/chrome-browser-ignoring-autocomplete-off/34998771#34998771
What happened:
The autoComplete attribute is still "off". It seems it's not possible to override it.

Reproduction repository:

Problem description:

Suggested solution:
Because of changes in the Chrome implementation, it is necessary to be able to configure the value of the "autocomplete" attribute.

**Comment by denkristoffer** [comment]
Setting `autoComplete` on the `<Downshift />` component itself doesn't do anything, but you can override it by setting it in `getInputProps()`, like so:

```jsx
<Downshift>
  {({
    ...
    getInputProps,
    ...
  }) => (
    <input
      {...getInputProps({
        ...
        autoComplete: 'off',
        ...
      })}
    />
  )}
</Downshift>
```

(It's been a while since I looked at Downshift's API, please excuse me if this has changed)

**Comment by alexandernanberg** [comment]
Closing this as the original issue was solved by the [2.0.12 release](https://github.com/paypal/downshift/releases/tag/v2.0.12). 

Doing it like @denkristoffer suggests would work in earlier versions of Downshift also.
